packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -186,7 +187,12 @@ const nextServerlessLoader: loader.Loader = function() {
         } catch (err) {
           console.error(err)
           await onError(err)
-          res.statusCode = 500
+
+          if (err.code === 'DECODE_FAILED') {
+            res.statusCode = 400
+          } else {
+            res.statusCode = 500
+          }
           res.end('Internal Server Error')
```

Comment:
This message seems wrong when we have a status code of 400.

Reply:
Updated to respond with `Bad Request` like we would show if we rendered the `_error` page